docker/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM python:3.7 as builder
+
+ARG MAKE_NUM_THREADS=4
+ARG OPENBLAS_NUM_THREADS=$MAKE_NUM_THREADS
+ENV MAKEFLAGS=" -j$MAKE_NUM_THREADS"
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
+    zlib1g-dev \
+    automake \
+    autoconf \
+    unzip \
+    git \
+    gfortran \
+    libtool \
+    subversion \
+    g++
+
+ADD gentle/ext /gentle/ext
+WORKDIR /gentle/ext
+
+# Compile Kaldi using OpenBLAS
+ARG OPENBLAS_COMMIT=tags/v0.3.7
```

Comment:
Make the version an evn var. Then we can set it in the circleCi config file.

Reply:
It is piped in from the ENV in the build.sh script